tests/unit/test_live_monitor.py

```
@@ -1,7 +1,113 @@
+            "main_session": MagicMock(session_id="session-ended", end_time=1000),
+            "all_sessions": [MagicMock(session_id="session-ended")],
+        }
+        active_workflow = {
+            "workflow_id": "session-active",
+            "main_session": MagicMock(session_id="session-active", end_time=None),
+            "all_sessions": [MagicMock(session_id="session-active")],
+        }
+
+        call_count = [0]
+
+        def mock_get_workflows(self, db_path):
```

Ruff (0.15.1)

[warning] Unused function argument: `self`

(ARG001)